### PR TITLE
Update user guide to align with editLoan command changes.

### PR DESCRIPTION
**Pull request by Neethesh26**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -268,9 +268,9 @@ Format: `editLoan INDEX VALUE REASON`
 
 Examples:
 
-* `editLoan 2 30 bought logistics`
-* `editLoan 3 -30 return money from logisitics`
-* `list` followed by `editLoan 1 -20 return money` will edit the 1st person in the SectresBook,
+* `editLoan 2 amt/30 reason/bought logistics`
+* `editLoan 3 amt/-30 reason/return money from logisitics`
```
Comment (rui-han-crh) [inline docs/UserGuide.md:272]: Wait sorry, recently i inserted the ability to edit by name with the same method, can you also add
`editLoan alex amt/-30 reason/return money from logistics` to indicate either edit by INDEX <OR> NAME

**Review by rui-han-crh — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (Neethesh26) [reply docs/UserGuide.md]: Resolved please check!

**Review by rui-han-crh — APPROVED** [review]
LGTM

**Review by Pinran-J — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 2]
```diff
@@ -464,7 +464,7 @@ Action | Format
 **Edit** | `edit INDEX [name/NAME] [phone/PHONE_NUMBER] [email/EMAIL] [home/ADDRESS] [bday/BIRTHDAY][tag/TAG]…​` | `edit 2 name/James Lee email/jameslee@example.com`
 **Find** | `find KEYWORD [MORE_KEYWORDS]` <br> `find NUMBER`                                                    | `find James Jake` <br> `find 8651`
 **Find Tag** | `findTag TAG [MORE_TAGS]`                                                                            | `findTag Operations Outreach`
-**Edit Loan** | `editLoan INDEX AMOUNT REASON`                                                                       | `editLoan 1 -20 Buy Logistics`
+**Edit Loan** | `editLoan INDEX amt/AMOUNT reason/REASON` <br> `editLoan NAME amt/AMOUNT reason/REASON`                | `editLoan 1 amt/-20 reason/Buy Logistics` <br> `editLoan alex amt/30 reason reason/payment`
```
Comment (ryanczx) [inline docs/UserGuide.md:467]: Extra "reason" in `editLoan alex amt/30 reason reason/payment`

**Comment by czhongwei** [comment]
closes #132 
closes #142

File: docs/UserGuide.md (hunk 2 above)
Comment (Neethesh26) [reply docs/UserGuide.md]: Fixed please check

**Pull request merged**